README.md

```diff
@@ -2,173 +2,160 @@
+      * [Contributing to the Kogito Operator](#contributing-to-the-kogito-operator)
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Requirements
+## Kogito Operator requirements
 
-- go v1.12
-- [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) v0.11.0
-- ocp 3.11/4.x (you can use [CRC](https://github.com/code-ready/crc) for local deployment)
-- [kogito s2i imagestreams](https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml) installed
+- Go v1.12 is installed.
+- The [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) v0.11.0 is installed.
+- OpenShift 3.11 or 4.x is installed. (You can use [CRC](https://github.com/code-ready/crc) for local deployment.)
+- The [kogito s2i imagestreams](https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml) is installed.
 
-## Installation
+## Kogito Operator installation
 
-### Deploy to OpenShift 4.x
+### Deploying to OpenShift 4.x
 
-Kogito operator is a namespaced operator, which means that you will need to install it into the namespace you want your Kogito application to run in.
+The Kogito operator is a namespaced operator, so you must install it into the namespace where you want your Kogito application to run.
 
-#### Optional Step
-
-You can import the Kogito image stream using the `oc client` manually with the following command:
+(Optional) You can import the Kogito image stream using the `oc client` manually with the following command:
 
 ```bash
 $ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
 ```
 
-But, this step is not mandatory anymore, when kogito is going to install a new app, it will create the required imagestreams
-before.
+This step is optional because Kogito creates the required imagestreams when it installs a new application.
 
-#### Via OperatorHub automatically
+#### Automatically in OperatorHub
 
-The installation on OpenShift 4.x is pretty straightforward since Kogito Operator is available in the OperatorHub as a community operator. The Operator can be easily found by filtering by _Kogito_ name.
+The Kogito Operator is available in the OperatorHub as a community operator. To find the Operator, search by the _Kogito_ name.
 
-Just follow the OpenShift Web Console instructions in the _Catalog_, _OperatorHub_ section in the left menu to install it in any namespace in the cluster. 
+You can also verify the Operator availability in the catalog by running the following command:
+
+```bash
+$ oc describe operatorsource.operators.coreos.com/kogitocloud-operator -n openshift-marketplace
+```
+Follow the OpenShift Web Console instructions in the **Catalog** -> **OperatorHub** section in the left menu to install it in any namespace in the cluster.
 
 ![Kogito Operator in the Catalog](docs/img/catalog-kogito-operator.png?raw=true)
```

Comment:
Should we update the screenshot ? Or wait for version 1.0 ?

Reply:
let’s wait for 1.0